### PR TITLE
feat: add additive manufacturing configuration

### PR DESCRIPTION
**Pull request by kpm-berlin**

### CODE REVIEW

File: configuration.json.example [hunk 1]
```diff
@@ -106,6 +110,21 @@
...
        }
+    },
+    {
+       "Namespace":"http://opcfoundation.org/UA/AdditiveManufacturing/NodeSet2/",
+       "Types":[
+         {
+           "Uri":"http://opcfoundation.org/UA/AdditiveManufacturing/NodeSet2/",
+           "Id":"i=1031",
+           "$comment":"AdditiveManufacturingType"
+         },
```
[the remark this answers is not in the document]
Comment (GoetzGoerisch) [reply configuration.json.example:121]: You also need to add the IdentificationType definiton analog to MachineToolType.

```
      "IdentificationType": {
        "Uri": "http://opcfoundation.org/UA/MachineTool/",
        "Id": "i=11",
        "$comment": "MachineToolIdentificationType"
      }
```

[the remark this answers is not in the document]
Comment (kpm-berlin) [reply configuration.json.example:121]: I think I did this below, no? 
But this is outside the `Types` Object

File: configuration.json.example [hunk 2]
```diff
@@ -106,6 +110,21 @@
...
+       ],
+       "IdentificationType":{
+         "Uri":"http://opcfoundation.org/GMS/",
```
Comment (GoetzGoerisch) [inline configuration.json.example:124]: URI shall be that of the AM CS.

File: configuration.json.example (hunk 1 above)
[the remark this answers is not in the document]
Comment (kpm-berlin) [reply configuration.json.example:121]: resolved, but are there other types needed in this `Types` section here besides the `AdditiveManufacturingType`?

**Comment by GoetzGoerisch** [comment]
@kpm-berlin please rebase. Good to merge then.
Thank you for your contribution.

**Review by GoetzGoerisch — APPROVED** [review]

**Pull request merged**